runtime/themes/serika-light.toml

```diff
@@ -52,6 +52,9 @@
 "ui.selection" = { fg = "bg0", bg = "bg3" }
 "ui.virtual.whitespace" = { fg = "bg2" }
 "ui.virtual.ruler" = { bg = "bg01" }
+"ui.virtual.inlay-hint" = { fg = "grey2", modifiers = ["italic"] }
+"ui.virtual.inlay-hint.parameter" = { fg = "grey2", modifiers = ["italic"] }
+"ui.virtual.inlay-hint.type" = { fg = "grey2", modifiers = ["italic"] }
```

Comment:
Why not just keep only `ui.virtual.inlay-hint`?

Reply:
🤔 this is unintentional. I wanted to try styling it differently at first. Can push an update after lunch.